src/Cartridge.php

```diff
@@ -80,7 +80,7 @@ public static function showMassiveActionsSubForm(MassiveAction $ma)
                 $input = $ma->getInput();
                 $input['maxpages'] = isset($input['maxpages']) ? (int) $input['maxpages'] : '';
 
-                echo "<input type='text' name='pages' value=\"" . $input['maxpages'] . "\" size='6'>";
+                echo "<input type='text' name='pages' value=\"" . htmlescape($input['maxpages']) . "\" size='6'>";
```

Comment:
Seems useless, it's already casted as int on line 81

Reply:
Indeed, the problem with Psalm is that it consider this as a security issue because value can be a string `''` and it does not seems to check if the string is sane. This change was here to fix a false positive issue.

This is why I am still not convinced that it is the most appropriate tool to use.

Reply:
No idea for paslm... I've tried to run it on a smaller project, but it never ends  :D

For this false positive, we can keep a "local" fix, but cast as int rather than using htmlescape.

Reply:
Casting an empty string as int would result in `value="0"` instead of `value=""`. I was not sure about the functional impact, so keeping the current code is the best option for now.

Reply:
Indeed, I forgot about empty string.